### PR TITLE
Add delivery_zone_name and outside_delivery_area to the accepted orde…

### PR DESCRIPTION
**Pull request by jolurodr**
Add fields delivery_zone_name and outside_delivery_area to the Accepted Orders API

### CODE REVIEW

File: accepted_orders/README.md [hunk 1]
```diff
@@ -131,67 +131,69 @@ The fields of an order collection are:
...
...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```
Comment (sinfield057) [inline accepted_orders/README.md]: Type should be `boolean or null`

File: accepted_orders/README.md [hunk 2]
```diff
@@ -131,67 +131,69 @@ The fields of an order collection are:
...
```
Comment (sinfield057) [inline accepted_orders/README.md]: Type should be `string or null`

File: accepted_orders/README.md (hunk 1 above)
Comment (jolurodr) [reply accepted_orders/README.md]: Fixed

File: accepted_orders/README.md (hunk 2 above)
Comment (jolurodr) [reply accepted_orders/README.md]: Fixed

**Review by sinfield057 — APPROVED** [review]
Looks good.

**Pull request merged**